assets/sass/globals/typography.scss

```diff
@@ -10,8 +10,7 @@ h1,
     font-weight: 600;
     @include font-and-leading(18px, 26px);
 
-    // @TODO: Add more granular named breakpoints
-    @media screen and (min-width: 380px) {
+    @media screen and (min-width: xsmall) {
```

Comment:
Is this some CSS magic or is it an error? eg. does it know what `xsmall` means?!

Reply:
Urm… it's sadly an error. A magic error? 💥